app/subscriber/src/features/filter-media/FilterMediaLanding.tsx

```diff
@@ -0,0 +1,146 @@
+import { useFilterOptions } from 'components/navbar';
+import { PageSection } from 'components/section';
+import React from 'react';
+import { useContent } from 'store/hooks';
+import { Col, ISourceModel, Row, Show } from 'tno-core';
+
+import { MediaFilterTypes } from './constants';
+import { FilterMedia } from './FilterMedia';
+import * as styled from './styled';
+import { alphabetArray } from './utils';
+
+/** The FilterMediaLanding Consists of the filter to narrow down the results of media, as well as the results. It is returned in a two column layout*/
+export const FilterMediaLanding: React.FC = () => {
+  const {
+    dailyPrint,
+    sources,
+    weeklyPrint,
+    cpWire,
+    talkRadio,
+    onlinePrint,
+    television,
+    newsRadio,
+  } = useFilterOptions();
```

Comment:
I have refactored `useFilterOptions` away in my latest PR.  It wasn't being used anywhere else so I moved it inline with the component.  Might need to move it back out... I'd be happy to assist.